Unable to run it on heroku

Hey!

I was able to run the example on my local machine but i have tried everything on Heroku but it doesn't work. I can access the parse server from my local machine without any problem.
I change from https to http vice versa.
I double checked all var.
This is the log from heroku:

2017-04-20T05:40:48.998084+00:00 heroku[web.1]: Starting process with command `npm start`
2017-04-20T05:40:51.066930+00:00 app[web.1]: 
2017-04-20T05:40:51.066955+00:00 app[web.1]: > parse-dashboard-example@1.0.2 start /app
2017-04-20T05:40:51.066955+00:00 app[web.1]: > node index.js
2017-04-20T05:40:51.066956+00:00 app[web.1]: 
2017-04-20T05:40:51.152807+00:00 app[web.1]: /app/index.js:8
2017-04-20T05:40:51.152826+00:00 app[web.1]:   let localParseServer = 'http://localhost:1337/parse';
2017-04-20T05:40:51.152827+00:00 app[web.1]:       ^^^^^^^^^^^^^^^^
2017-04-20T05:40:51.152828+00:00 app[web.1]: SyntaxError: Unexpected identifier
2017-04-20T05:40:51.152829+00:00 app[web.1]:     at createScript (vm.js:53:10)
2017-04-20T05:40:51.152829+00:00 app[web.1]:     at Object.runInThisContext (vm.js:95:10)
2017-04-20T05:40:51.152830+00:00 app[web.1]:     at Module._compile (module.js:543:28)
2017-04-20T05:40:51.152830+00:00 app[web.1]:     at Object.Module._extensions..js (module.js:580:10)
2017-04-20T05:40:51.152831+00:00 app[web.1]:     at Module.load (module.js:488:32)
2017-04-20T05:40:51.152832+00:00 app[web.1]:     at tryModuleLoad (module.js:447:12)
2017-04-20T05:40:51.152832+00:00 app[web.1]:     at Function.Module._load (module.js:439:3)
2017-04-20T05:40:51.152833+00:00 app[web.1]:     at Module.runMain (module.js:605:10)
2017-04-20T05:40:51.152833+00:00 app[web.1]:     at run (bootstrap_node.js:423:7)
2017-04-20T05:40:51.152834+00:00 app[web.1]:     at startup (bootstrap_node.js:147:9)
2017-04-20T05:40:51.159642+00:00 app[web.1]: 
2017-04-20T05:40:51.162507+00:00 app[web.1]: npm ERR! Linux 3.13.0-112-generic
2017-04-20T05:40:51.162890+00:00 app[web.1]: npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "start"
2017-04-20T05:40:51.163301+00:00 app[web.1]: npm ERR! node v7.9.0
2017-04-20T05:40:51.163520+00:00 app[web.1]: npm ERR! npm  v4.2.0
2017-04-20T05:40:51.163781+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2017-04-20T05:40:51.164018+00:00 app[web.1]: npm ERR! errno 1
2017-04-20T05:40:51.164167+00:00 app[web.1]: npm ERR! parse-dashboard-example@1.0.2 start: `node index.js`
2017-04-20T05:40:51.164288+00:00 app[web.1]: npm ERR! Exit status 1
2017-04-20T05:40:51.164426+00:00 app[web.1]: npm ERR! 
2017-04-20T05:40:51.164562+00:00 app[web.1]: npm ERR! Failed at the parse-dashboard-example@1.0.2 start script 'node index.js'.
2017-04-20T05:40:51.164671+00:00 app[web.1]: npm ERR! Make sure you have the latest version of node.js and npm installed.
2017-04-20T05:40:51.164805+00:00 app[web.1]: npm ERR! If you do, this is most likely a problem with the parse-dashboard-example package,
2017-04-20T05:40:51.165058+00:00 app[web.1]: npm ERR! not with npm itself.
2017-04-20T05:40:51.165178+00:00 app[web.1]: npm ERR! Tell the author that this fails on your system:
2017-04-20T05:40:51.165295+00:00 app[web.1]: npm ERR!     node index.js
2017-04-20T05:40:51.165422+00:00 app[web.1]: npm ERR! You can get information on how to open an issue for this project with:
2017-04-20T05:40:51.165552+00:00 app[web.1]: npm ERR!     npm bugs parse-dashboard-example
2017-04-20T05:40:51.165676+00:00 app[web.1]: npm ERR! Or if that isn't available, you can get their info via:
2017-04-20T05:40:51.165786+00:00 app[web.1]: npm ERR!     npm owner ls parse-dashboard-example
2017-04-20T05:40:51.165919+00:00 app[web.1]: npm ERR! There is likely additional logging output above.
2017-04-20T05:40:51.182943+00:00 app[web.1]: 
2017-04-20T05:40:51.183160+00:00 app[web.1]: npm ERR! Please include the following file with any support request:
2017-04-20T05:40:51.183299+00:00 app[web.1]: npm ERR!     /app/.npm/_logs/2017-04-20T05_40_51_167Z-debug.log
2017-04-20T05:40:51.291333+00:00 heroku[web.1]: State changed from starting to crashed
2017-04-20T05:40:51.283750+00:00 heroku[web.1]: Process exited with status 1


Does anyone is having the same problem with heroku?

--- a/app.json
+++ b/app.json
@@ -26,7 +26,7 @@
       "value": "stefanayala3266@gmail.com"
     },
     "PASSWORD": {
-      "description": "The password of the account to log into the dashboard.",
+      "description": "The password of the account to log into your dashboard.",
       "value": "parse"
     }
   },

--- a/app.json
+++ b/app.json
@@ -20,6 +20,14 @@
     "APP_NAME": {
       "description": "The name of the app that will show up in the dashboard.",
       "value": "myAppName"
+    },
+    "USERNAME": {
+      "description": "The username of the account to log into the dashboard.",
+      "value": "stefanayala3266@gmail.com"
+    },
+    "PASSWORD": {
+      "description": "The password of the account to log into the dashboard.",
+      "value": "parse"
     }
   },
   "image": "heroku/nodejs"

--- a/index.js
+++ b/index.js
@@ -16,7 +16,7 @@ var dashboard = new ParseDashboard({
       serverURL: process.env.SERVER_URL || herokuParseServer || localParseServer,
       appName: process.env.APP_NAME || 'MyApp',
       users: [
-        { user: process.env.USERNAME, pass: process.env.PASSWORD }
+        { user: "username", pass: "password" }
       ],
       trustProxy: 1
     },

--- a/index.js
+++ b/index.js
@@ -21,6 +21,10 @@ var dashboard = new ParseDashboard({
       trustProxy: 1
     },
   ],
+  users: [
+    { user: "username", pass: "password" }
+  ],
+  trustProxy: 1
 });
 
 var app = express();

--- a/index.js
+++ b/index.js
@@ -4,12 +4,11 @@ var express = require('express');
 var ParseDashboard = require('parse-dashboard');
 var path = require('path');
 
-var dashboard = new ParseDashboard({
-  let localParseServer = 'http://localhost:1337/parse';
-
+let localParseServer = 'http://localhost:1337/parse';
   // Heroku requires HTTPS. Please read the README file for details.
-  // let herokuParseServer = 'https://my-parse-dashboard.herokuapp.com/parse'
+let herokuParseServer = 'https://my-parse-dashboard.herokuapp.com/parse';
 
+var dashboard = new ParseDashboard({
   apps: [
     {
       appId: process.env.APP_ID || 'myAppId',

--- a/index.js
+++ b/index.js
@@ -3,6 +3,7 @@
 var express = require('express');
 var ParseDashboard = require('parse-dashboard');
 var path = require('path');
+var config = require('./config.json');
 
 let localParseServer = 'http://localhost:1337/parse';
   // Heroku requires HTTPS. Please read the README file for details.
@@ -15,6 +16,7 @@ var dashboard = new ParseDashboard({
       masterKey: process.env.MASTER_KEY || 'myMasterKey',
       serverURL: process.env.SERVER_URL || herokuParseServer || localParseServer,
       appName: process.env.APP_NAME || 'MyApp',
+      users: config.users
     },
   ],
 });

--- a/index.js
+++ b/index.js
@@ -15,14 +15,10 @@ var dashboard = new ParseDashboard({
       masterKey: process.env.MASTER_KEY || 'myMasterKey',
       serverURL: process.env.SERVER_URL || herokuParseServer || localParseServer,
       appName: process.env.APP_NAME || 'MyApp',
-      users: [
-        { user: "username", pass: "password" }
-      ],
-      trustProxy: 1
     },
   ],
   users: [
-    { user: "username", pass: "password" }
+    { user: process.env.USERNAME, pass: process.env.PASSWORD }
   ],
   trustProxy: 1
 });

--- a/index.js
+++ b/index.js
@@ -3,7 +3,6 @@
 var express = require('express');
 var ParseDashboard = require('parse-dashboard');
 var path = require('path');
-var config = require('./config.json');
 
 let localParseServer = 'http://localhost:1337/parse';
   // Heroku requires HTTPS. Please read the README file for details.
@@ -16,7 +15,9 @@ var dashboard = new ParseDashboard({
       masterKey: process.env.MASTER_KEY || 'myMasterKey',
       serverURL: process.env.SERVER_URL || herokuParseServer || localParseServer,
       appName: process.env.APP_NAME || 'MyApp',
-      users: config.users
+      users: [
+        { user: process.env.USERNAME, pass: process.env.PASSWORD }
+      ]
     },
   ],
 });

--- a/index.js
+++ b/index.js
@@ -17,7 +17,8 @@ var dashboard = new ParseDashboard({
       appName: process.env.APP_NAME || 'MyApp',
       users: [
         { user: process.env.USERNAME, pass: process.env.PASSWORD }
-      ]
+      ],
+      trustProxy: 1
     },
   ],
 });

